vendor/wp-now/src/wp-now.ts

```diff
@@ -614,6 +614,27 @@ async function mountInternalMuPlugins( php: PHP, options: WPNowOptions ) {
 			`
 		);
 	}
+
+	// Intercept wp_mail and send email data via Electron service.
+	// @TODO If we need dynamic ports, think about bundling this in the site folder.
+	php.writeFile(
+		path.posix.join( PLAYGROUND_INTERNAL_MU_PLUGINS_FOLDER, '0-wp-mail.php' ),
+		`<?php
+			add_filter( 'wp_mail', function( $args ) {
+				$response = wp_remote_post( 'http://localhost:7777/email', array(
```

Comment:
TODO: the port number should be dynamic.

It's set in the app.

- PHP const?
- WP option?
- ...

Reply:
I'm thinking this should be copied to `/mu-plugins/` for each separate site.

This would simplify setting internal constants for the port number.

Also, users might want use a customized or other local email server.